spec/features/submit_story_spec.rb

```diff
@@ -134,7 +134,7 @@
   end
 
   scenario "submitting a tracking link" do
```

Comment:
```suggestion
  scenario "submitting a banned domain" do
```

Just updating the name to reflect that this test is more general now.